pyproject.toml

```
@@ -8,7 +8,7 @@ name = "ansys-tools-path"
 version = "0.3.dev0"
 description = "Library to locate Ansys products in a local machine."
 readme = "README.rst"
-requires-python = ">=3.7"
+requires-python = ">=3.8"
```

```suggestion
requires-python = ">=3.8,<4"
```

Set upper limit to Python 4 (whenever it comes)